pkgs/development/perl-modules/generic/default.nix

```diff
@@ -17,6 +17,8 @@ toPerlModule(stdenv.mkDerivation (
 
     checkTarget = "test";
 
+    LINKTYPE = if stdenv.targetPlatform.isStatic then "static" else "dynamic";
```

Comment:
Worth a note in https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/perl.section.md

Reply:
Fixed.